debezium-core/src/main/java/io/debezium/pipeline/metrics/SnapshotChangeEventSourceMetrics.java

```diff
@@ -100,9 +100,9 @@ public void monitoredTablesDetermined(Iterable<TableId> tableIds) {
     }
 
     @Override
-    public void tableSnapshotCompleted(TableId tableId, long numRows) {
-        rowsScanned.put(tableId.toString(), numRows);
-        remainingTables.remove(tableId.toString());
+    public void dataCollectionSnapshotCompleted(String dataCollectionId, long numRows) {
```

Comment:
Could this be `DataCollectionId` instead of `String`?

Reply:
It could though I would suggest adding a method to `DataCollectionId` rather than relying on `toString()`.  Wdyt?

Reply:
Ok, works for me 👍

Reply:
Added this as a part of commit 62123b6.